### ISSUE TITLE
Add "Date added" and "Chef" columns.

### ISSUE THREAD

**Issue by kadishmal** [garethr/vagrantboxes-heroku#74]
Right now when downloading several of the Vagrant boxes from http://www.vagrantbox.es/, I found out only after running them that Chef in them are outdated and doesn't support the software stack I need to run.

So, it would be nice to have additional columns on the page which tell when a particular box was added and what version of Chef does it include. This will help us avoid unnecessary downloads of large files.

Each contributor should fill out these columns when contributing their Vagrant boxes.
